tempto-core/src/main/java/io/prestosql/tempto/internal/hadoop/hdfs/HdfsModuleProvider.java

```diff
@@ -55,10 +60,10 @@ public Module getModule(Configuration configuration)
             protected void configure()
             {
                 Set<String> configurationKeys = configuration.listKeys();
-                if (!configurationKeys.contains(CONF_HDFS_WEBHDFS_HOST_KEY)
+                if (!configurationKeys.contains(CONF_HDFS_WEBHDFS_URI_KEY)
                         || !configurationKeys.contains(CONF_TESTS_HDFS_PATH_KEY)) {
-                    logger.debug("No HDFS support enabled as '{}' or '{}' is not configured",
-                            CONF_HDFS_WEBHDFS_HOST_KEY,
+                    logger.debug("No HDFS support enabled as '{}' or '{}' not is configured",
```

Comment:
you swapped `not` with `is`, please undo

Reply:
thanks